File: Sully.c
#include <stdio.h>
#include <string.h>

main() {
int i=5;
while(i>=0){
char file_name[80]="Sully_";
char n=48+i;
strncat(file_name, &n, 1);
strcat(file_name, ".c");
FILE *fp=fopen(file_name, "w+");
char *a = "#include <stdio.h>%c#include <string.h>%c%cmain() {%cint i=%d;%cwhile(i>=0){%cchar file_name[80]=%cSully_%c;%cchar n=48+i;%c";
char *b = "strncat(file_name, &n, 1);%cstrcat(file_name, %c.c%c);%cFILE *fp=fopen(file_name, %cw+%c);%cchar *a = %c%s%c;%cchar *b = %c%s%c;%c";
char *c = "char *c = %c%s%c;%cfprintf(fp, a, 10, 10, 10, 10, i, 10, 10, 34, 34, 10, 10);%cfprintf(fp, b, 10, 34, 34, 10, 34, 34, 10, 34, a, 34, 10, 34, b, 34, 10);%cfprintf(fp, c, 34, c, 34, 10, 10, 10, 10, 10, 10, 10);%ci--;%c}%c}%c";
fprintf(fp, a, 10, 10, 10, 10, i, 10, 10, 34, 34, 10, 10);
fprintf(fp, b, 10, 34, 34, 10, 34, 34, 10, 34, a, 34, 10, 34, b, 34, 10);
fprintf(fp, c, 34, c, 34, 10, 10, 10, 10, 10, 10, 10);
i--;
}
}

File: Sully.py
i = 5
while i >= 0:
	file_name = "Sully_" + str(i) + ".py"
	fp = open(file_name, "w+")
	s = "i = %d%cwhile i >= 0:%c%cfile_name = %cSully_%c + str(i) + %c.py%c%c%cfp = open(file_name, %cw+%c)%c%cs = %c%s%c%c"
	fp.write(s % (i, 10, 10, 9, 34, 34, 34, 34, 10, 9, 34, 34, 10, 9, 34, s, 34, 10))
	cs = "%cfp.write(s %c (i, 10, 10, 9, 34, 34, 34, 34, 10, 9, 34, 34, 10, 9, 34, s, 34, 10))%c%ccs = %c%s%c%c%cfp.write(cs %c (9, 37, 10, 9, 34, cs, 34, 10, 9, 37, 10, 9, 10))%c%ci -= 1%c"
	fp.write(cs % (9, 37, 10, 9, 34, cs, 34, 10, 9, 37, 10, 9, 10))
	i -= 1
